src/main/java/com/uber/cadence/internal/sync/POJOActivityTaskHandler.java

```diff
@@ -186,6 +188,9 @@ public Result handle(
           metricsScope,
           isLocalActivity);
     }
+    if (metricsRateLimiter.tryAcquire(1)) {
+      metricsScope.gauge(MetricsType.ACTIVITY_ACTIVE_THREAD_COUNT).update(Thread.activeCount());
```

Comment:
Add a tag to indicate local v.s. normal activity.

Reply:
I use a different metrics type for local activity to match the existing metrics naming pattern.
ACTIVITY_ACTIVE_THREAD_COUNT is the thread count of normal activity.
LOCAL_ACTIVITY_ACTIVE_THREAD_COUNT is the thread count of local activity.